src/Runner.Common/Constants.cs

```diff
@@ -146,9 +146,10 @@ public static class ReturnCode
                 public const int RunOnceRunnerUpdating = 4;
             }
 
-            public static class Features
+            public static class FeatureFlags
```

Comment:
Perhaps we could consider putting future FFs in this class? It's probably preferable over literals.